bolt-sidecar/src/driver.rs

```diff
@@ -122,7 +124,7 @@ impl SidecarDriver<StateClient, PrivateKeySigner> {
         let keystore_signer = SignerBLS::Keystore(keystore);
 
         // Commitment responses are signed with a regular Ethereum wallet private key.
-        let commitment_key = opts.commitment_private_key.0.clone();
+        let commitment_key = opts.commitment_opts.operator_private_key.0.clone();
```

Comment:
fwiw, i like the new naming, commitment_opts.operator_private_key is very clear, nice improvement!